e2e_tests/jest.config.js

```diff
@@ -29,6 +29,6 @@ module.exports = {
             },
         },
     },
-    transformIgnorePatterns: ['/node_modules/(?!react-virtualized).+\\.js$', '_utils/websocket.js'],
+    transformIgnorePatterns: ['/node_modules/(?!@enykeev/react-virtualized).+\\.js$', '_utils/websocket.js'],
```

Comment:
Since the [previous library ](https://github.com/bvaughn/react-virtualized) is not compatible with react-17 , I've replaced it with the forked version. 
please check the below comment: 
https://github.com/bvaughn/react-virtualized/commit/005be24a608add0344284053dae7633be86053b2#commitcomment-75710883